agent/util-scripts/register-tool-set

```diff
@@ -94,9 +95,20 @@ default_tool_set=$(getconf.py -l default-tool-set pbench/tools)
 case "$toolset" in
 	default)
 	for i in $default_tool_set; do
-		register-tool --name=$i $remote $label $group -- --interval="$interval"
+		if [ ! -z $remote ] ; then
+			# ensure pbench packages are installed on remote system 
+			ssh $ssh_opts $(echo $remote | cut -d"=" -f2) yum -y install "$pbench_agent"
```

Comment:
@ndokos same as in #113.With this change we would install pbench on remote system only if we want to collect data from it ( if register-tool-set invoked with register-tool-set --remote ) your opinion is welcome